ext/crypto/lib.rs

```diff
@@ -876,11 +876,17 @@ pub async fn op_crypto_derive_bits(
       let salt = hkdf::Salt::new(algorithm, salt);
       let prk = salt.extract(&secret);
       let info = &[&*info];
-      let okm = prk.expand(info, HkdfOutput(length))?;
```

Comment:
Cleaner way to avoid `match`
```rust
map_err(|e| {
  custom_error("DOMExceptionOperationError", "...")
})?
```

Reply:
that indeed looks much cleaner.

```rust
let okm = prk.expand(info, HkdfOutput(length)).map_err(|_e| {
        custom_error(
          "DOMExceptionOperationError",
          "The length provided for HKDF is too large",
        )
      })?;
```

Reply:
Hi,

Thank you for suggesting this change; please check new revision.

Thank you,
Upendra Upadhyay.